v1/contracts/Store.sol

```diff
@@ -0,0 +1,68 @@
+/*
+  Store implementation.
+ 
+  An implementation of StoreInterface with a fee per second and withdraw functions for the owner.
+*/
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./StoreInterface.sol";
+import "./Withdrawable.sol";
+import "./FixedPoint.sol";
+
+
+// An implementation of StoreInterface that can accept Oracle fees in ETH or any arbitrary ERC20 token.
+contract Store {
+
+    using SafeMath for uint;
+    using FixedPoint for FixedPoint.Unsigned;
+    using FixedPoint for uint;
+
+    uint private fixedOracleFeePerSecond; // Percentage of 10^18. E.g., 1e18 is 100% Oracle fee.
+    uint private constant FP_SCALING_FACTOR = 10**18;
+
+    uint private constant WEEKLY_DELAY_FEE = 0; //<-- governance vote?
```

Comment:
I think we probably want this to be similar to `fixedOracleFeePerSecond` (set by some specific address managed by `MultiRole`). If we were to add governance, we would just give a governance contract permission to change this.